api/api.go

```diff
@@ -33,18 +35,20 @@ type KalaStatsResponse struct {
 
 // HandleKalaStatsRequest is the hanlder for getting system-level metrics
 // /api/v1/stats
-func HandleKalaStatsRequest(w http.ResponseWriter, r *http.Request) {
-	resp := &KalaStatsResponse{
-		Stats: job.NewKalaStats(),
-	}
+func HandleKalaStatsRequest(cache job.JobCache) func(w http.ResponseWriter, r *http.Request) {
```

Comment:
@ajvb You don't need to include Kala in the name. It's implied by the namespace.